README.md

```diff
@@ -49,6 +49,19 @@ COPY your-application.war ROOT.war
 The Tomcat instance can be configured through the environment variable `TOMCAT_PROPERTIES` which is
 a comma-separated list of `name=value` pairs appended to `catalina.properties`.
 
+### Stackdriver Logging
+When the Tomcat runtime is running on Google App Engine, all output to stdout/stderr is captured by Stackdriver 
+and is available via the Stackdriver logging console.
+ 
+However more detailed and integrated logs are available if the Stackdriver logging mechanism is used directly.
```

Comment:
This is too vague to decide whether to use it. See https://github.com/GoogleCloudPlatform/jetty-runtime#enhanced-stackdriver-logging-beta for a better description.